cpp/include/Ice/Connection.h

```diff
@@ -144,23 +142,23 @@ namespace Ice
         /// @return The connection information.
         [[nodiscard]] virtual ConnectionInfoPtr getInfo() const = 0;
 
-        /// Set the connection buffer receive/send size.
-        /// @param rcvSize The connection receive buffer size.
-        /// @param sndSize The connection send buffer size.
+        /// Sets the size of the connection received and send buffers.
```

Comment:
why received and send? shouldn't be both in the same tense?

Reply:
Yes, should be `receive`.